code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+    int n , max , tmp;
+    cout << "Enter numbers of elements : ";
```

Comment:
Instead of asking for the size, use a std::vector, which reads elements until a EOF/EOL, and constantly check for a bigger number.